src/wasimock.rs

```diff
@@ -86,14 +104,14 @@ pub extern "C" fn __syscall_getcwd(_buf: i32, _size: i32) -> i32 {
 #[no_mangle]
 //pub extern "C" fn __wasi_environ_sizes_get(_environ_count: libc::size_t, _environ_buf_size: libc::size_t) -> i32 {
 pub extern "C" fn __wasi_environ_sizes_get(_environ_count: i32, _environ_buf_size: i32) -> i32 {
-    env::log_str("__wasi_environ_sizes_get");
+    //env::log_str("__wasi_environ_sizes_get");
```

Comment:
did you leave these commented on purpose?

Reply:
Good catch ! Yes I had to leave these commented on purpose, because otherwise I got problems when building with Rust 1.81.0. The issue is brought up here: https://github.com/rust-lang/rust/issues/130210